### PR TITLE
Setup private chain

### PR DESCRIPTION
**Pull request by thanhnguyennguyen**

### CODE REVIEW

File: docs/developers/tutoprivatenet.md [hunk 1]
```diff
@@ -4,4 +4,170 @@ so that one can learn to contribute to the development of TomoChain.
 The tutorial is alo a first step for any developer who wants to customize TomoChain's 
 source code to use in a private or consortium environment.
 
-## Content to be updated
\ No newline at end of file
+- Install Golang
```
Comment (thanhson1085) [inline docs/developers/tutoprivatenet.md]: It should be `## Install Golang`

Comment (thanhson1085) [reply docs/developers/tutoprivatenet.md]: heading 2

File: docs/developers/tutoprivatenet.md [hunk 2]
```diff
@@ -4,4 +4,166 @@ so that one can learn to contribute to the development of TomoChain.
...
+    --keystore $HOME/tomochain/keystore \
+    --password [YOUR_PASSWORD_FILE_TO_LOCK_YOUR_ACCOUNT]
+   ```
+    
+## Customize genesis block
+   - Run puppeth command and answer questions about your private chain
+   ```
+    puppeth
+   ```
+   ![Private chain name](https://user-images.githubusercontent.com/17243442/57121919-bcbbd000-6da4-11e9-8a0e-dea3a15f3fc1.png)
```
Comment (thanhson1085) [inline docs/developers/tutoprivatenet.md]: should use text, friendly for SEO and copp/paste text

File: docs/developers/tutoprivatenet.md [hunk 3]
```diff
@@ -4,4 +4,166 @@ so that one can learn to contribute to the development of TomoChain.
...
+   ![POSV configurations](https://user-images.githubusercontent.com/17243442/57122012-2f2cb000-6da5-11e9-8b1e-7fc1c034226a.png)
+   ![MultiSig wallet setting](https://user-images.githubusercontent.com/17243442/57122031-453a7080-6da5-11e9-92d6-49fba3a4c1ea.png)
```
Comment (thanhson1085) [inline docs/developers/tutoprivatenet.md]: Wrong inputs. Should use the wallets that we controls private key. Then we can unlock multisig wallet

**Comment by thanhnguyennguyen** [comment]
move to https://github.com/tomochain/docs/pull/151

**Pull request closed without merging**